crawler.py

```diff
@@ -12,8 +12,8 @@
 import time
 from shutil import copytree
 from urllib.request import urlopen
+from tranco import Tranco
```

Comment:
Do we need to update requirements.txt? Wouldn't running Docker fail otherwise since there is no `tranco` package present?

Reply:
Ah good catch, thank you! I had this on my local machine but didn't commit that file. Added to next commit.